snap/snapcraft.yaml

```diff
@@ -3,315 +3,105 @@ summary: Live streaming and video recording software
-      - libvorbisfile3
-      - libvpl2
-      - libvpx7
-      - libwayland-client0
-      - libwayland-cursor0
-      - libwayland-egl1
-      - libwayland-server0
-      - libwebp7
-      - libwebpmux3
+      - libva-drm2
+      - libvlc5
+      - libvlc5
       - libx11-6
-      - libx11-xcb1
-      - libx264-163
-      - libx265-199
-      - libxau6
+      - libx264-164
+      - libxcb1
       - libxcb-composite0
       - libxcb-randr0
-      - libxcb-render0
-      - libxcb-shape0
       - libxcb-shm0
       - libxcb-xfixes0
       - libxcb-xinerama0
-      - libxcb1
-      - libxcomposite1
-      - libxcursor1
-      - libxdamage1
-      - libxdmcp6
-      - libxext6
-      - libxfixes3
-      - libxi6
-      - libxinerama1
       - libxkbcommon0
-      - libxml2
-      - libxrandr2
-      - libxrender1
-      - libxss1
-      - libxv1
-      - libxvidcore4
-      - libxxf86vm1
-      - libzimg2
-      - libzmq5
-      - libzstd1
-      - libzvbi0
-      - pipewire-bin
-      - ocl-icd-libopencl1
-      - zlib1g
+      - pulseaudio
+      - python3
+      - python3.12
       - qt6-image-formats-plugins
       - qt6-qpa-plugins
       - qt6-wayland
-      - gstreamer1.0-plugins-good
-      - gstreamer1.0-plugins-bad
-      - gstreamer1.0-plugins-ugly
-      - gstreamer1.0-x
-      - libgles2-mesa
-      - libvlc5
-      - vlc-plugin-base
       - stterm
-  cleanup:
-    after: [obs, ndi, launcher]
-    plugin: nil
-    build-snaps: [gnome-42-2204]
-    override-prime: |
-      set -eux
-      cd /snap/gnome-42-2204/current
-      find . -type f,l -exec rm -f $CRAFT_PRIME/{} \;
+      - vlc-plugin-base
+
 apps:
   obs-studio:
-    common-id: com.obsproject.Studio
-    extensions: [gnome]
     command: obs-wrapper
+    extensions: [gnome]
     environment:
-      QT_PLUGIN_PATH: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/qt6/plugins:$QT_PLUGIN_PATH
-      LD_LIBRARY_PATH: $LD_LIBRARY_PATH:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/lapack:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/openblas-pthread:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/blas:$SNAP/bin/64bit:$SNAP/obs-plugins/64bit:$SNAP/data/obs-scripting/64bit:$SNAP/cef:$SNAP/lib
-      PIPEWIRE_CONFIG_NAME: $SNAP/usr/share/pipewire/pipewire.conf
-      PIPEWIRE_MODULE_DIR: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pipewire-0.3
-      SPA_PLUGIN_DIR: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/spa-0.2
+      QT_PLUGIN_PATH: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/qt6/plugins:$QT_PLUGIN_PATH
+      QT_QPA_PLATFORM_PLUGIN_PATH: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/qt6/plugins/platforms
+      LD_LIBRARY_PATH: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/libproxy:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/pulseaudio:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/obs-plugins:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/obs-scripting:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/blas:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/lapack:$LD_LIBRARY_PATH
+      PATH: $SNAP/ffmpeg-platform/usr/bin:$PATH
```

Comment:
This should only be needed if you are using the `ffmpeg-2404` plug, for example: https://github.com/snapcrafters/audacity/blob/candidate/snap/snapcraft.yaml#L62-L65

If you do introduce this plug (which I think is a good idea), you should also remove `ffmpeg` from `stage-packages` and update `LD_LIBRARY_PATH` to find the mounted ffmpeg libs.

Reply:
true, thanks!